client/my-sites/checkout/controller.jsx

```diff
@@ -52,8 +52,10 @@ export function checkout( context, next ) {
 	const isDisallowedForSitePicker =
 		context.pathname.includes( '/checkout/no-site' ) &&
 		( isLoggedOut || ! hasSite || isDomainOnlyFlow );
+	const isJetpackCheckout = context.pathname.includes( '/checkout/jetpack' );
```

Comment:
Not a blocker, but I'm wondering if we should rename `isJetpackCheckout` to `isJetpackUserlessCheckout` to be more clear that this flag is used for _userless_ checkout specifically, and not for the checkout of all things Jetpack?

Reply:
I think that makes sense. Feel free.